tools/gen_docs.py

```diff
@@ -128,13 +128,13 @@ def test_hash(verbose=False, debug=False):
     match = result == original
     if not match:
         print("FAIL: Internal documents are outdated! Please update via \"python tools/gen_docs.py\"")
-    return int(not match)
```

Comment:
`int` is the desired return as we are returning the value `0` or `1` as the exit code, as any numerical, non zero value will fail the test.